.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, windows-latest, macos-latest]
+        os: [windows-11-arm, ubuntu-24.04, windows-latest, macos-latest]
```

Comment:
when our release_pipelines are working, I would suggest to "comment out" windows-11-arm here (not adapting the other changes in that file and try to fix the "main"-pipeline separatley? I know that's probably not "clean"

Reply:
@justinchuby 
would     
continue-on-error: ${{ matrix.os == 'windows-11-arm' }}
ok for you?

Reply:
Either way (commenting or skipping like you suggested) works for me!

Reply:
I added skipping